packs/data/feats.db/diverse-lore.json

```diff
@@ -19,7 +19,24 @@
         "prerequisites": {
             "value": []
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "skill-ckeck",
+                "key": "RollOption",
+                "label": "Diverse Lore",
+                "option": "diverse-lore-active",
+                "toggleable": true
+            },
+            {
+                "key": "FlatModifier",
+                "label": "Diverse Lore",
+                "predicate": [
+                    "diverse-lore-active"
```

Comment:
```suggestion
                    "diverse-lore"
```

Reply:
Another question: Would this not lead to the penalty always being applied as it checks for `diverse-lore`, which is the slug of the feat?


Sorry if this is much ado about such a tiny change, but I'm trying to learn the general principles at work here.

Reply:
A roll option for a feat always starts with "feat:"